### ISSUE TITLE
Toolbar script still loads when toolbar is disabled

### ISSUE THREAD

**Issue by paolodamico** [PostHog/posthog#8799]
## Bug description

A [user](https://app.papercups.io/conversations/all/491e74f5-de0f-4308-b859-d01432dac8c2) reported this bug where even after disabling the toolbar we still load the `toolbar.js` script. This adds network overhead plus uses resources on the local device.

![d0465dcc95474752b8d5d07392cccabe-2022-02-25_10-37_1](https://user-images.githubusercontent.com/5864173/156265632-c83d782f-35f5-40be-8f2c-cb7d2ce0d297.png)

## Expected behavior

If the toolbar is disabled, don't load this script at all.

## How to reproduce

1. Disable the toolbar from the project settings.
2. See the network requests tab.

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

**Comment by pauldambra** [comment]
When we successfully load the toolbar we write the public token into `_posthogEditorParams` (along with some other info) in local storage.

When you load the page, if there's a token in that value, we load the toolbar without waiting for the decide response. 

Just need to decide the sensible way to unpick that...

**Comment by pauldambra** [comment]
🤔  Also, when toolbar is disabled. If you visit the /toolbar page and click Launch toolbar we redirect to your site with all the config to launch the toolbar. Despite the toolbar being disabled.

**Comment by pauldambra** [comment]
@clarkus what should we show on this page when the toolbar feature is disabled?

![Screenshot 2022-03-17 at 11 08 32](https://user-images.githubusercontent.com/984817/158796504-fe17c7d1-21b1-420a-8c7e-7fea28ffc32a.png)

A call to action to visit settings like when recordings is off, presumably?

![Screenshot 2022-03-17 at 11 10 22](https://user-images.githubusercontent.com/984817/158796796-dda5c4ac-db20-46ea-af61-606fe0e376cb.png)

**Comment by pauldambra** [comment]
feels like three things here:

- [ ] posthog-js should respect the decide endpoint result
- [ ] toolbar UI should not present the launch toolbar button
- [ ] backend should not redirect with toolbar credentials if toolbar is disabled

**Comment by mariusa** [comment]
+ stop loading the toolbar.js script. This adds network overhead (2MB from toolbar + css) plus uses resources on the local device.

**Comment by pauldambra** [comment]
> stop loading the toolbar.js script. 

Yep @mariusa, absolutely. That's step 1 "respect the endpoint result"

Currently once we've stored a token for the toolbar, we never check again. You can reproduce by clearing `_posthogEditorParams` in local storage after disabling toolbar in settings. We don't load the toolbar js then

**Comment by mariusa** [comment]
Indeed, thanks!

**Comment by pauldambra** [comment]
we removed the concept of disabling the toolbar, you can clear storage if you want to remove the toolbar from a browser instance
